### DISCUSSION TITLE
cmake --build fails on include <bit>  (ver 6.0.0+0)

### DISCUSSION THREAD

**Discussion by baruchgu** [MikePopoloski/slang#1053] [Q&A]
Help, please with the build and install 
I have dounloaded .ZIP file, unzip.
..
cmake -B build
cmake --build build -j8
..
if fails in include <bit>

 21%] Building CXX object source/CMakeFiles/slang_slang.dir/AllSyntax.cpp.o
In file included from xxxx/slang-4221f2fa18e6e532bb38124c409c93703dff7999/source/../include/slang/parsing/Token.h:10,
                 from xxxx/slang-4221f2fa18e6e532bb38124c409c93703dff7999/build/source/slang/syntax/AllSyntax.h:10,
                 from xxxx/slang-4221f2fa18e6e532bb38124c409c93703dff7999/build/source/AllSyntax.cpp:8:
xxxx/slang-4221f2fa18e6e532bb38124c409c93703dff7999/source/../include/slang/numeric/SVInt.h:10:10: fatal error: bit: No such file or directory
 #include <bit>
          ^~~~~
compilation terminated.
gmake[2]: *** [source/CMakeFiles/slang_slang.dir/build.make:151: source/CMakeFiles/slang_slang.dir/AllSyntax.cpp.o] Error 1

**Comment by MikePopoloski** [comment]
You need to have a C++20 compatible build environment set up on your system in order to build the library.
